src/ext/conversion/__test__/import-properties.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { Align, ColumnWidthType, ExportFormat } from '../../../types';
+import importProperties, { getColumnInfo, getMultiColumnInfo } from '../import-properties';
+
+describe('importProperties', () => {
+  const qDimensions = [
+    {
+      qDef: {
+        textAlign: {
+          auto: true,
+          align: 'left' as Align,
+        },
+        columnWidth: {
+          type: 'auto' as ColumnWidthType,
+          pixels: 200,
+          percentage: 20,
+        },
+      },
+      qAttributeExpressions: [],
+    },
+  ];
+  const qMeasures = [
+    {
+      qDef: {
+        qDef: '',
+        textAlign: {
+          auto: true,
+          align: 'left' as Align,
+        },
+        columnWidth: {
+          type: 'auto' as ColumnWidthType,
+          pixels: 200,
+          percentage: 20,
+        },
+      },
+      qAttributeExpressions: [],
+    },
+  ];
+
+  describe('getColumnInfo', () => {
+    const index = 0;
+
+    it('should get the correct columnInfo given columnWidths with -1', () => {
+      const columnWidths = [-1, -1];
+      let newColumnInfo = getColumnInfo(qDimensions, index, columnWidths);
+      expect(newColumnInfo.qDef.columnWidth).toEqual({ type: 'fitToContent' });
+
+      newColumnInfo = getColumnInfo(qMeasures, 1, columnWidths, 1);
+      expect(newColumnInfo.qDef.columnWidth).toEqual({ type: 'fitToContent' });
+    });
+
+    it('should get the correct columnInfo given columnWidths with pixels', () => {
+      const columnWidths = [300, 20];
+      let newColumnInfo = getColumnInfo(qDimensions, index, columnWidths);
+      expect(newColumnInfo.qDef.columnWidth).toEqual({ pixels: 300, type: 'pixels' });
+
+      newColumnInfo = getColumnInfo(qMeasures, 1, columnWidths, 1);
+      expect(newColumnInfo.qDef.columnWidth).toEqual({ pixels: 20, type: 'pixels' });
+    });
+
+    it('should get the correct columnInfo given columnWidths with -1 and pixels', () => {
+      const columnWidths = [400, -1];
+      let newColumnInfo = getColumnInfo(qDimensions, index, columnWidths);
+
+      expect(newColumnInfo.qDef.columnWidth).toEqual({ pixels: 400, type: 'pixels' });
+
+      newColumnInfo = getColumnInfo(qMeasures, 1, columnWidths, 1);
+      expect(newColumnInfo.qDef.columnWidth).toEqual({ type: 'fitToContent' });
+    });
```

Comment:
I don't see the point of having three tests here. You are basically doing the first two in the third test. Which you might as well test in `getMultiColumnInfo`